pkg/agent/proxy/proxier.go

```diff
@@ -82,12 +83,18 @@ type Proxier interface {
 	GetServiceByIP(serviceStr string) (k8sproxy.ServicePortName, bool)
 }
 
-type proxier struct {
-	once                sync.Once
```

Comment:
why was it needed for `proxier` but not `ProxyServer`?

Reply:
I think this is not needed by proxier. It's only used here:

```
func (p *proxier) Run(stopCh <-chan struct{}) {
	p.once.Do(func() {
		p.ofClient.RegisterPacketInHandler(uint8(openflow.PacketInCategorySvcReject), p)
		go p.serviceConfig.Run(stopCh)
		if p.endpointSliceEnabled {
			go p.endpointSliceConfig.Run(stopCh)
			if p.topologyAwareHintsEnabled || p.serviceTrafficDistributionEnabled {
				go p.nodeConfig.Run(stopCh)
			}
		} else {
			go p.endpointsConfig.Run(stopCh)
		}
		p.stopChan = stopCh
		p.SyncLoop()
	})
}
```

The above snippet Is only called once in `cmd/antrea-agent/agent.go`. I don't think this will be called twice when starting antrea-agent. Did I get it right?